appserver/pom.xml

```diff
@@ -111,7 +111,7 @@
         <dbschema.version>6.6</dbschema.version>
         <schema2beans.version>6.6</schema2beans.version>
         <mq.version>5.1.2-b01</mq.version>
-        <derby.version>10.13.1.1</derby.version>
+        <derby.version>10.13.1.2</derby.version>
```

Comment:
Is this official derby release? I can't see it in [maven central](http://repo1.maven.org/maven2/org/apache/derby/derby/).
Why not upgrade to 10.14.2.0?

Reply:
> It comes out of [repackaged](https://github.com/javaee/repackaged/tree/master/derby) project which is packaging derby 10.14.2.0. It is released as "org.glassfish.external:derby"
thanks for update

I found it a bit confusing (10.13 vs. 10.14). [It wouldn't be if versions were much, much different, and not so similar :wink:.)

But you are definitely right, [derby-10.13.1.2.pom](http://repo1.maven.org/maven2/org/glassfish/external/derby/10.13.1.2/derby-10.13.1.2.pom) references 10.14.2.0.
Thanks for the reply.